src/app/search/facet-search/facet-search-update.pipe.ts

```diff
@@ -0,0 +1,28 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'getFacetSearchUpdate',
+})
+/**
+ * This pipe updates the number of hidden items in the facet when a search is completed
+ *
+ * @param {Array<any>} items The items in the facet
+ * @param {string} searchText The search text entered
+ * @returns {string} The numbers of hidden items in the facet
+ * @memberof GetFacetSearchResultsPipe
+ */
+export class GetFacetSearchUpdatePipe implements PipeTransform {
+  transform(unfilteredItems: Array<any>, searchText: string): string {
+    let hiddenFilteredSize;
+    if (!searchText && unfilteredItems.length > 5) {
```

Comment:
Magic number 5 used 3 times in this method; should extract to a const. I thought our linter caught stuff like this.

Reply:
Only our Java checkstyle (not tests) seems to complain about magic numbers.  Javascript/Typescript linters/projects typically don't have that as a default unless otherwise specified

Reply:
Can confirm I configured it for Java non-tests. 
Would be nice for the UI (non-tests) as well